site/posts/jupyter-notebooks-faq.ipynb

```diff
@@ -0,0 +1,195 @@
+    "\n",
+    "-----"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "**Q.** The Jupyter workspace autocompletion of quotation marks and parenthesis is causing errors. Is there a way to disable this?\n",
+    "\n",
+    "**A.** Yes according to this this [stack overflow post](https://stackoverflow.com/questions/44216326/how-to-disable-auto-quotes-and-auto-brackets-in-jupyter-5-0#:~:text=2%20Answers&text=After%20executing%20the%20Python%20command,auto%2Dclosing%20quotes%20and%20brackets), in any code cell type: \n",
+    "\n",
+    "```\n",
+    "from notebook.services.config import ConfigManager\n",
+    "c = ConfigManager()\n",
+    "c.update('notebook', {\"CodeCell\": {\"cm_config\": {\"autoCloseBrackets\": False}}})\n",
+    "```\n",
+    "\n",
+    "You may have to restart the Jupyter session for the changes to take effect.\n",
+    "\n",
+    "-----"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "**Q.** Is converting a cell from code to raw a good way to basically “comment out” an entire cell?  Is that good practice or bad practice?\n",
+    "\n",
+    "**A.** This is the easiest way to \"comment out\" a cell.\n",
+    "\n",
+    "-----"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "**Q.** Equations aren’t working in my markdown cells. What should I do?\n",
+    "\n",
+    "**A.** Double check that you've used `$$` to bookend your equation.\n",
+    "\n",
+    "-----"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "**Q.** How do I use Jupyter with an R kernel (instead of Python)?\n",
+    "\n",
+    "**A.** You would need to create a new Conda environment with R and `jupyter` installed.\n",
+    "\n",
+    "-----"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "**Q.** Does Jupyter lab care about being shut down gracefully?  If I you just `ctrl-C` it from the command-line, does that do anything bad?\n",
+    "\n",
+    "**A.** As long as your notebook is saved, `ctrl-C` is fine. Otherwise, you will lose the contents of your notebooks that weren’t saved before the shutdown.\n",
+    "\n",
+    "-----"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "**Q.** What’s advantage of using Jupyter Notebook, as compared to working in the terminal?\n",
+    "\n",
+    "**A.** Jupyter is better for creating a narrative. It allows you to alternate between markdown text explanations, data visualization plots right after the plot calls, and analysis code as you go through the workflow. This tends to be easier to follow than having all of the output at once when the script is run.\n",
+    "\n",
+    "-----"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
```

Comment:
This is also not a specific question.  The real answer is "watch the video," but that's also not helpful.  If a person asks this question before actually seeing the video, then the answer is "Wait for it!"  And if they have seen the video and ask this question, then the answer is "It's your preference."

```suggestion
```